submission_fetcher.py

```diff
@@ -0,0 +1,71 @@
+import random as r
+import shutil
+import requests
+
+import praw
+
+reddit = praw.Reddit(client_id='AmJbMF6Sh56JNg',
+                             client_secret='1K1ovK2ku9EP_usjLR-8CFRiQB8',
+                             user_agent='windows:test:v0.1 (by u/unclesam79)')
```

Comment:
Probably not a good idea to share this with the world. Add to .gitignore

Reply:
> Probably not a good idea to share this with the world. Add to .gitignore

oh SHIT

Reply:
Credentials have been revoked and new credentials have been shifted to a gitignored file.